core/src/main/java/org/mskcc/cbio/portal/scripts/ImportCopyNumberSegmentData.java

```diff
@@ -144,7 +147,17 @@ public void run() {
     
     private void importFractionGenomeAltered(CancerStudy cancerStudy) throws DaoException {
 
-        DaoCopyNumberSegment.calculateFractionGenomeAltered(cancerStudy.getInternalId());
+        ClinicalAttribute clinicalAttribute = DaoClinicalAttributeMeta.getDatum(FRACTION_GENOME_ALTERED_ATTR_ID, cancerStudy.getInternalId());
+        if (clinicalAttribute == null) {
+            ClinicalAttribute attr = new ClinicalAttribute(FRACTION_GENOME_ALTERED_ATTR_ID, "Fraction Genome Altered", "Fraction Genome Altered", "NUMBER",
+                false, "1", cancerStudy.getInternalId());
+            DaoClinicalAttributeMeta.addDatum(attr);
+        }
+        
+        Map<Integer, String> mutationCounts = DaoCopyNumberSegment.getFractionGenomeAltered(cancerStudy.getInternalId());
```

Comment:
Maybe create a function `DaoMutation.createFractionGenomeAlteredClinicalData(cancerStudyId)` and moved lines 150-160 there? It maybe also good to make `DaoCopyNumberSegment.getFractionGenomeAltered` private or even move the code inside `createFractionGenomeAlteredClinicalData`.